<ui version="4.0" >
 <class>AddEditBlogDialog</class>
 <widget class="QDialog" name="AddEditBlogDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic" >
      <attribute name="title" >
       <string>&amp;Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>&amp;Configuring your blog account...</string>
         </property>
         <property name="buddy" >
          <cstring>txtUrl</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Blog/Homepage &amp;URL :</string>
         </property>
         <property name="buddy" >
          <cstring>txtUrl</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="txtUrl" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>User&amp;name :</string>
         </property>
         <property name="buddy" >
          <cstring>txtUser</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QLineEdit" name="txtUser" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>&amp;Password :</string>
         </property>
         <property name="buddy" >
          <cstring>txtPass</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2" >
        <widget class="QLineEdit" name="txtPass" />
       </item>
       <item row="4" column="2" >
        <widget class="QPushButton" name="btnAutoConf" >
         <property name="text" >
          <string>&amp;Auto Configure</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced" >
      <attribute name="title" >
       <string>A&amp;dvanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>&amp;Configuring your blog account...</string>
         </property>
         <property name="buddy" >
          <cstring>txtUrl</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>API :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="comboAPI" />
       </item>
       <item row="1" column="2" >
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>195</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Blog ID :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="txtID" />
       </item>
       <item row="2" column="2" >
        <widget class="QPushButton" name="btnFetch" >
         <property name="maximumSize" >
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>Fetch ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>130</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEditBlogDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEditBlogDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
